### PR TITLE
Gives the Gas Station a Teleporter so most people can actually get there! Also antiteleporter implants and gps signal!

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑
tweak: Gas station now has a teleporter so the average person can now actually go there to buy things!
/🆑

### CODE REVIEW

**Comment by ThatLing** [comment]
Why does this need a teleporter? Isn't the entire premise of the gas station that you're supposed to find it when space exploring?

**Comment by fluffe9911** [comment]
Cause its a lavaland player ruin whose whole goal is to sell things to people who pass by and its pretty hard to do when the majority of people dont have easy access to your place.

edit also think your getting this confused with some other ruin lol

**Comment by ThatLing** [comment]
Wait, this is a lavaland ruin. Why are you adding a teleporter to a lavaland ruin, wtf

**Comment by fluffe9911** [comment]
Because theirs a ghost role there who is supposed to sell things to people who pass by the problem is that getting people to actually come there to buy stuff is a bloody pain cause its location is random and the only people otherwise who can actually access it are miners, this is just to help make things easier so the general public can actually buy things from them

long story short: ghost role sell things, people cant get there to buy things, lets people get there to buy things.

Response to below: Nu

**Comment by ThatLing** [comment]
Just remove the ghost role instead

**Comment by SomeguyManperson** [comment]
Just add a GPS with maybe a custom name or a tracking beacon if it's really important otherwise the gas station guy is just gonna teleport to the station first chance they get/miners get a free teleporter

**Comment by fluffe9911** [comment]
although then it would be a one way trip which is ehhhhhhhhhh I dont really know what to do honestly

**Review by Amelia0010 — DISMISSED** [review]
someguymanperson is right, if they get a teleporter they'll just teleport back to the station immediately. See: space bartender

**Comment by fluffe9911** [comment]
Mk will change it later

**Comment by Altoids1** [comment]
Are you gonna rename this PR some or...

File: yogstation/code/game/objects/items/implants/implant_teleporter.dm [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+/obj/item/implant/teleporter/removed(mob/living/source, silent = 0, special = 0)
+	..()
+	say("Implant tampering detected.")
+	source.gib()
+
+/obj/item/implant/teleporter/ghost_role
+	name = "employee retrieval implant"
+	usewhitelist = TRUE
+	retrievalmessage = "Employee retrieval complete."
```
Comment (Altoids1) [inline yogstation/code/game/objects/items/implants/implant_teleporter.dm:101]: I'm personally against coding in "Do not leave" for ghostroles.

Comment (Amelia0010) [reply yogstation/code/game/objects/items/implants/implant_teleporter.dm:101]: I'm not :)

Comment (ThatLing) [reply yogstation/code/game/objects/items/implants/implant_teleporter.dm:101]: Same

Comment (Amelia0010) [reply yogstation/code/game/objects/items/implants/implant_teleporter.dm:101]: Same as me or same as Altoids

Comment (ThatLing) [reply yogstation/code/game/objects/items/implants/implant_teleporter.dm:101]: Yes

**Comment by ThatLing** [comment]
Agree with nich's agree, make this a GPS

**Comment by fluffe9911** [comment]
@ThatLing I already resolved this with nich. Gas station clerk has a cant leave z level implant now so theirs no issue with him having a teleporter now ye meme

edit: also I already added a gps signal as well

**Comment by ThatLing** [comment]
Why a teleporter AAAAAAAAAAAAA

**Comment by fluffe9911** [comment]
@ThatLing SO PEOPLE CAN GO TO THE GAS STATION TO BUY THE GAS STATION CLERKS GOODS THEN LEAVE WHEN THEY ARE DONE REEEEE

**Review by monster860 — APPROVED** [review]

**Pull request merged**